src/main/java/seedu/nutrigoals/logic/commands/SetupCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.nutrigoals.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.nutrigoals.logic.parser.CliSyntax.PREFIX_GENDER;
+import static seedu.nutrigoals.logic.parser.CliSyntax.PREFIX_HEIGHT;
+import static seedu.nutrigoals.logic.parser.CliSyntax.PREFIX_IDEAL_WEIGHT;
+import static seedu.nutrigoals.logic.parser.CliSyntax.PREFIX_WEIGHT;
+
+import seedu.nutrigoals.logic.commands.exceptions.CommandException;
+import seedu.nutrigoals.model.Model;
+import seedu.nutrigoals.model.user.User;
+
+
+/**
+ * Sets up user profile in NutriGoals
+ */
+public class SetupCommand extends Command {
+
+    public static final String COMMAND_WORD = "setup";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets up the users profile for the tracker. "
+            + "Parameters: "
+            + PREFIX_GENDER + "Gender "
+            + PREFIX_HEIGHT + "Height "
+            + PREFIX_WEIGHT + "Weight"
+            + PREFIX_IDEAL_WEIGHT + "Ideal Weight"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_GENDER + "M "
+            + PREFIX_HEIGHT + "170 "
+            + PREFIX_WEIGHT + "70"
+            + PREFIX_IDEAL_WEIGHT + "65";
```

Comment:
Perhaps it'll be better to add new line characters so that the message is more readable. 

Currently, it looks like this:
<img width="852" alt="Screenshot 2022-10-13 at 12 56 50 AM" src="https://user-images.githubusercontent.com/97275050/195403364-59f2f036-c6ea-4979-ad74-49cf8a806db9.png">

```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets up the users profile for the tracker.\n"
            + "Parameters: "
            + PREFIX_GENDER + "Gender "
            + PREFIX_HEIGHT + "Height "
            + PREFIX_WEIGHT + "Weight "
            + PREFIX_IDEAL_WEIGHT + "Ideal Weight\n"
            + "Example: " + COMMAND_WORD + " "
            + PREFIX_GENDER + "M "
            + PREFIX_HEIGHT + "170 "
            + PREFIX_WEIGHT + "70 "
            + PREFIX_IDEAL_WEIGHT + "65";
```